Experimental/conda/environment.yml

```diff
@@ -0,0 +1,11 @@
+name: fab
+dependencies:
+  - python=3.8
```

Comment:
Is there a way in the syntax here to specify a minimum version instead of fixing to `3.8`?  (and if there is, would that allow someone to use this environment/recipe to build against a Python version of their choice provided it is recent enough?)

Reply:
Yes, the [docs seem to allow](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#package-match-specifications) that. Not sure what our minimum spec is. I know Dave's got things running with 3.7 since his vdi is not playing ball, so we could use `>=3.7` or possible even just `=3`.

Reply:
Ah, however, the python and pip versions need to match...